src/qos_client/Cargo.toml

```diff
@@ -15,15 +15,16 @@ qos_nsm = { path = "../qos_nsm", default-features = false }
 ureq = { version = "2.9", default-features = false }
 aws-nitro-enclaves-nsm-api = { version = "0.4", default-features = false }
 borsh = { version = "1.0", features = ["std", "derive"] , default-features = false}
-p256 = { version = "0.12.0", default-features = false }
+p256 = { version = "0.13.0", default-features = false }
 rand_core = { version = "0.9", features = ["os_rng"], default-features = false }
 zeroize = { version = "1.6", default-features = false }
 rpassword = { version = "7", default-features = false }
 serde_json = { version = "1" }
 
+der = { version = "0.7", default-features = false, optional = true }
 x509 = { version = "0.2", default-features = false, optional = true }
-# As of 7/2025, upgrading this to 0.8.0 is blocked due to dependency conflicts
-yubikey = { version = "0.7", features = ["untested"], default-features = false, optional = true }
+x509-cert = { version = "0.2", default-features = false, optional = true }
```

Comment:
we can do this in a separate PR, but now that we are using `x509-cert`, can we get away with removing `x509` crate? Seems like unnesecary dependency review burden if we can get away with only one of these